apps/web/src/lang/fr.json

```diff
@@ -6857,5 +6857,45 @@
   "s7Hnlg": {
     "defaultMessage": "Modifier<hidden> l’équipe</hidden>",
     "description": "Breadcrumb title for the edit team page link."
+  },
+  "2kXthI": {
+    "defaultMessage": "Rôles des équipes",
+    "description": "Heading for updating a users team roles"
+  },
+  "3IZ3mN": {
+    "defaultMessage": "Équipe",
+    "description": "Title displayed for the role table display team column"
+  },
+  "GaMSN8": {
+    "defaultMessage": "Équipe",
+    "description": "Label for the input to select team of a team role"
+  },
```

Comment:
Not an issue strictly related to this PR but we need to exercise a little caution with re-translating things multiple times. The biggest issue with re-translating things that should be the same is different translations coming back resulting in inconsistent wording at the French site. 

![image](https://user-images.githubusercontent.com/40485260/227646217-172569d3-0e8a-4b64-a131-0eb1c80ea55b.png)

Reply:
Yeah, I know this has come up a few times.  We tend to err on the side of caution and separately translate the same word multiple times and up with a lot of duplicates.  In theory, the same word in English could be different words in French in different contexts but you have to know the language to know if this is case.  :smile:   And you could up with contradictions, as you mentioned.